src/praisonai-agents/test_retry_fix.py

```
@@ -0,0 +1,86 @@
+    
+    print("✅ ToolExecutionError.error_category attribute test passed!")
+    return True
+
+def test_retry_config_validation():
+    """Test ToolRetryConfig validation."""
+    from praisonaiagents.config import ToolRetryConfig
+    
+    # Test validation of max_attempts
+    try:
+        config = ToolRetryConfig(max_attempts=0)
+        assert False, "Should have raised ValueError"
```

Ruff (0.15.18)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)